From 9ef211dcb89edbad605c49964faf302fb97b6ba9 Mon Sep 17 00:00:00 2001
From: Guy Shapiro <guy.shapiro@mobi-wize.com>
Date: Thu, 24 Aug 2017 10:58:14 +0300
Subject: [PATCH] Disable hard link check by default

Without the "-p" flag, cronie does several checks on the
crontab files. Those checks prevent crontrab files from being
editable by unauthorized users.
One of those checks won't permit loading file that is
hard link count greater than 1.

When OSTree manages crontab file, it uses hard links.
All those user checks are irrelevant in the case embedded system, so
disable them by default.

Signed-off-by: Guy Shapiro <guy.shapiro@mobi-wize.com>
---
 crond.sysconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crond.sysconfig b/crond.sysconfig
index ee23703..1cb083f 100644
--- a/crond.sysconfig
+++ b/crond.sysconfig
@@ -1,3 +1,3 @@
 # Settings for the CRON daemon.
 # CRONDARGS= :  any extra command-line startup arguments for crond
-CRONDARGS=
+CRONDARGS="-p"
-- 
2.1.4

